packages/react/accordion/src/Accordion.stories.tsx

```diff
@@ -292,6 +293,26 @@ export const Animated = () => {
           </AccordionItem>
         ))}
       </Accordion>
+
+      <h1>Open by default</h1>
+      <Accordion type="single" className={rootClass} value="One">
```

Comment:
nit: 
```suggestion
      <Accordion type="single" className={rootClass} defaultValue="One">
```

to bring it inline with the collapsible story so we can test subsequent open animations.

Reply:
Ah yes, good shout 👀